pr_agent/tools/pr_description.py

```diff
@@ -297,7 +297,7 @@ def _prepare_pr_answer(self) -> Tuple[str, str]:
                 value = self.file_label_dict
                 key_publish = "PR changes walkthrough"
             else:
-                key_publish = key.rstrip(':').replace("_", " ").capitalize()
+                key_publish = key.rstrip(':').replace("_", " ").title()
```

Comment:
i don't think this is good. 
i do want to capitalize only the first letter

Reply:
("PR changes walkthrough"
instead of
"PR Changes Walkthrough")

Reply:
Sure I will revert it, I just though since those are section titles it will be more fitting.